ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -69,7 +70,8 @@ type nodeService struct {
 }
 
 func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRequest) (*csi.NodeStageVolumeResponse, error) {
-	klog.V(4).InfoS("NodeStageVolume: called", "args", *req)
+	klog.InfoS("NodeStageVolume: called", "args", *req)
+	t := time.Now()
```

Comment:
nit: could you make the variable name a bit more descriptive?

Reply:
Done